server/Opendatacam.js

```diff
@@ -612,34 +612,30 @@ module.exports = {
   },
 
   sendUpdateToClient: function() {
-    // Stream it to client if SSE request is open
-    if(Opendatacam.sseConnexion) {
-      // Client connection could be established so log this once!
-      const isClosedOrUnknown = Opendatacam.isSseConnectionOpen == false || Opendatacam.isSseConnectionOpen == null;
-      if(isClosedOrUnknown) {
-        console.info('SSE: Sending update to the client');
-      }
-      Opendatacam.isSseConnectionOpen = true;
-
-      Opendatacam.sseConnexion(`data:${JSON.stringify({
-        trackerDataForLastFrame: Opendatacam.trackerDataForLastFrame,
-        counterSummary: this.getCounterSummary(),
-        trackerSummary: this.getTrackerSummary(),
-        videoResolution: Opendatacam.videoResolution,
-        appState: {
-          yoloStatus: Opendatacam.yolo ? Opendatacam.yolo.getStatus() : null,
-          isListeningToYOLO: Opendatacam.isListeningToYOLO,
-          recordingStatus: Opendatacam.recordingStatus
-        }
-      })}\n\n`);
-    } else {
-      // The client is not open. Log a info message, but only once!
-      const isOpenOrUnknown = Opendatacam.isSseConnectionOpen == true || Opendatacam.isSseConnectionOpen == null;
-      if(isOpenOrUnknown) {
-        console.info('SSE: Failed sending update to the client');
-      }
-      Opendatacam.isSseConnectionOpen = false;
+    const newValue = (Opendatacam.sseResponses.size > 0);
+    if (Opendatacam.isSseConnectionOpen === null || Opendatacam.isSseConnectionOpen !== newValue) {
+      // Log connection changes only once
+      console.info(newValue ?
+        'SSE: Sending update to the client' :
```

Comment:
Again should we indicate that this means no update could be sent at all?

Just looking at the console, if you don't know the code it could be mistaken as "a single client did (not) get the update"

Reply:
I've replaced the messages with: "SSE: Sending update to clients" and "SSE: All clients disconnected, cannot send update", does that sound better?